src/app/component/about-page/about-page.component.html

```diff
@@ -107,7 +116,7 @@ <h3 class="card-header">
         <span class="main-card-text" id="fifth-card-text">
           {{ 'about-us.steps.block-5.text' | translate }}
         </span>
-        <a class="card-link" href="" id="fifth-card-link">
+        <a class="card-link tertiary-global-button" href="" id="fifth-card-link">
           {{ 'about-us.steps.block-5.button' | translate }}
         </a>
       </div>
```

Comment:
generally put all attributes like class, href.. id and others in the column, if it more then 2 in one tag